Assets/MRTK/Core/Utilities/EyeGazeSmoother.cs

```diff
@@ -124,16 +124,16 @@ private bool IsSaccading(Ray rayOld, Ray rayNew)
                 {
                     Vector2 hv1 = new Vector2(v1.x, 0);
                     Vector2 hv2 = new Vector2(v2.x, 0);
-                    if (Vector2.Angle(hv1, hv2) > saccadeThreshInDegree)
+                    if (OnSaccadeX != null && Vector2.Angle(hv1, hv2) > saccadeThreshInDegree)
                     {
-                        PostOnSaccadeHorizontally();
+                        OnSaccade.Invoke();
```

Comment:
Looks like this is referencing the wrong event:

```suggestion
                        OnSaccadeX.Invoke();
```

Reply:
Whoops, good catch.